bot.py

```diff
@@ -338,6 +338,86 @@ async def display_current_list_of_summoners(ctx):
         await ctx.send(embed=create_embed(embed_data))
 
 
+@bot.command(name="teams", help="Display two teams")
+async def display_teams(ctx):
+    """Make and display teams to bot from list of summoners in json"""
+    try:
+        # typing indicator
+        async with ctx.typing():
+            await asyncio.sleep(1)
+
+        # server id
+        server_id = str(ctx.guild.id)
+
+        file_data = ""
+
+        if os.path.getsize(json_path) == 0 or not os.path.exists(json_path):
```

Comment:
Is this just a general exception? where we will tell customers 
```
"Oops! Something went wrong.\nTry again!"
```
Or can we tell them to register players?

Reply:
Yup, I can fix that

Reply:
Oh it's a general exception. If not people it'll catch NOT ENOUGH PLAYERS but I guess I can include
`To add player, Please add summoner by `@{bot.user.name} add``

Reply:
From what I see, since [here](https://github.com/dhmoon91/discord/pull/31/files#diff-36f6200cd8353111badb874140eba3af6055f4cc5839fbdeb9ede992620828c1R355) you raise a general exception, it will stop execution and go to https://github.com/dhmoon91/discord/pull/31/files#diff-36f6200cd8353111badb874140eba3af6055f4cc5839fbdeb9ede992620828c1R410 and will give error messages with 
```
error_title = f"{e_values}"
error_description = "Oops! Something went wrong.\nTry again!"
```
Have you tested this scenario of `if os.path.getsize(json_path) == 0 or not os.path.exists(json_path):`

Reply:
Oh you're right, I'll test the scenario and add if necessary.